longpath.inc

```diff
@@ -81,7 +81,7 @@ LONGPATH(CHAR_T *path)
 	*start = sep;
 	if (fhand != INVALID_HANDLE_VALUE) {
 	    STRLEN len = FN_STRLEN(fdata.cFileName);
-	    if ((STRLEN)(tmpbuf + sizeof(tmpbuf) - tmpstart) > len) {
+	    if ((STRLEN)(len < tmpbuf - tmpstart + sizeof(tmpbuf))) {
```

Comment:
I don't think the `(STRLEN)` cast is correct anymore in the rewritten condition, but I'll fix this in a separate commit.

Reply:
I've fixed this in https://github.com/perl-libwin32/win32/pull/18/commits/d1b3f62eb92694ef90fb416beffdadb5b9d9d1e6, but looking at this again, I think both expressions are identical; you swapped the left and right sides of the condition and changed `>` to `<`. Am I missing something? How does this fix a bufferoverflow?